packages/schema/src/config/generate.ts

```diff
@@ -14,11 +14,15 @@ export default defineUntypedSchema({
      * ```js
      * routes: ['/users/1', '/users/2', '/users/3']
      * ```
+     * 
+     * @type {string | string[]}
      */
     routes: [],
 
     /**
      * This option is no longer used. Instead, use `nitro.prerender.ignore`.
+     * 
+     * @deprecated
```

Comment:
Because we don't use this value, it felt appropriate to mark it as deprecated, but I'm happy to revert this change if we have plans for this prop later.